public/javascripts/Progress/src/Progress.js

```diff
@@ -51,30 +51,57 @@ function Progress (_, $, userRole) {
         });
     }
 
-    function putUserOrg(e) {
+    function putUserTeam(e, newTeam) {
         var parsedId = $(this).attr('id').split("-"); // the id comes in the form of "from-startOrg-to-endOrg"
-        var startOrg = parsedId[1];
-        var endOrg = parsedId[3];
+        var startTeam = parsedId[1];
+        if (newTeam != null) {
+            var endTeam = newTeam;
+        } else {
+            var endTeam = parsedId[3];
+        }
         $.ajax({
             async: true,
-            url: '/userapi/setUserOrg/' + endOrg,
+            url: '/userapi/setUserOrg/' + endTeam,
             type: 'put',
             success: function (result) {
                 window.location.reload();
-                if (endOrg != startOrg) {
-                    if (startOrg != 0) {
-                        logWebpageActivity("Click_module=leaving_org=" + startOrg);
-                    }
-                    if (endOrg != 0) {
-                        logWebpageActivity("Click_module=joining_org=" + endOrg);
-                    }
+                if (endTeam != startTeam) {
```

Comment:
Since you removed the current team from the list of teams to join, it looks like we don't need to have the `endTeam != startTeam` check anymore!